spec/controllers/users_controller_spec.rb

```diff
@@ -25,65 +25,19 @@
       let(:user_params) { Fabricate.attributes_for(:user) }
 
       before do
-        charge = double('charge', successful?: true)
-        StripeWrapper::Charge.should_receive(:create).and_return(charge)
+        result = double('result', successful?: true, message: "Successful!")
+        UserRegistration.any_instance.should_receive(:call).and_return(result)
```

Comment:
Should syntax is deprecated. The new syntax for this would be:

```ruby
expect_any_instance_of(UserRegistration).to receive(:call).and_return(result)
````